contrib/remote_hsmd/NOTES.md

```diff
@@ -21,22 +11,49 @@ On Fedora:
 
     sudo dnf install -y grpc-devel grpc-plugins
 
-Building
+Python packages:
+
+    pip3 install --user base58 bitstring secp256k1 mrkd
+
+    # In c-lightning root, remote-hsmd branch:
+    pip3 install --user -r requirements.txt
+
+    # Temporarily downgrade markupsafe to avoid breaking dependencies
+    pip3 install --user markupsafe==2.0.1
+
 
+Validating Lightning Signer:
+
+    # In parent directory of c-lightning root
+    git clone git@gitlab.com:lightning-signer/validating-lightning-signer.git
+    cargo build --features log_pretty_print,debug_enforcement_state
+
+C-Lightning:
+
+    # Add an "upstream" reference and fetch all tags (needed for pyln-{proto,client,testing})
+    # Use the HTTPS pointer if you have authentication issues with git
+    git remote add upstream git@github.com:ElementsProject/lightning.git
+    git fetch --all --recurse-submodules
+    git fetch --tags
```

Comment:
There's an impending merge conflict w/ https://github.com/lightning-signer/c-lightning/pull/51, I simplified/streamlined those two lines.  Should be easy to fix after whichever PR goes first though.

Reply:
Happy to rebase on top of #51 once it is merged.